model/ftn/w3uqckmd.ftn

```diff
@@ -877,7 +878,7 @@
 !/T1      WRITE (NDST,9011) NB0, 'CENTRAL'
 !
 !/OMPH/!$OMP PARALLEL DO PRIVATE (IP, IXY, CFL, IXYC, QB, IXYU, IXYD, &
-!/OMPH/!$OMP&                     DQ, DQNZ, QCN, QBN, QBR, CFAC )
+!/OMPH/!$OMP&                     DQ, DQNZ, QCN, QBN, QBR, CFAC, IY, IX )
```

Comment:
If switch T1 is enabled, variables QN, IX2 and IY2 are shared, not private (see around lines 909 - 911). I assume this will make the test output non-repoducable.

However, I don't know how you would deal with this; it requires an extra OMP directive that is enabled by both the T1 and OMPH switch, which we cannot do. This highlights a limitation of the WW3 compilation switches...

Reply:
@ukmo-ccbunney  I've added a mutually-exclusive condition in make_makefile.sh for OMPH and T1 due to conflict in w3uqckmd OpenMP loop. Also added note in manual switch.tex file. We could revisit this in case T1 is required in combination with OMPH (which at this point I think unlikely, FML).

Reply:
Changes in my previous comment (eg to make_makefile.sh and manual) have been reverted reflecting a more robust solution brought in via @ukmo-ccbunney changes to w3adv added to this branch/PR.